src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/designforextension/InputDesignForExtensionRequiredJavadocPhrase.java

```diff
@@ -0,0 +1,61 @@
+package com.puppycrawl.tools.checkstyle.checks.design.designforextension;
+
+/* Config:
+ * requiredJavadocPhrase = "^This implementation.*\\.$"
+ *
+ */
+public class InputDesignForExtensionRequiredJavadocPhrase {
+
+    /**
+     * This implementation is for <p> some html code
+     * </p>.
+     *
+     * @param a
+     * @param b
+     * @return sum
+     */
+    public int foo1(int a, int b) {return a + b;}  // ok, required comment pattern in javadoc
+
+    /**
+     * This implementation is required for ...
+     *
+     * @param a
+     * @param b
+     * @return sum
+     */
+    public int foo2(int a, int b) {return a + b;}  // ok, required comment pattern in javadoc
+
+    /** This implementation is for ... */
+    public int foo3(int a, int b) {return a + b;}  // ok, required comment pattern in javadoc
+
+    /**
+     * This implementation ...
+     */
+    public int foo4(int a, int b) {return a + b;}  // ok, required comment pattern in javadoc
+
+    /** This method can safely be overridden. */
```

Comment:
add a case of 
`/**This method can safely be overridden. */` if this is valid javadoc (I am not sure)

Reply:
![image](https://user-images.githubusercontent.com/31252532/105097628-edf79a80-5a76-11eb-9fab-310359faeb9f.png)




![image](https://user-images.githubusercontent.com/31252532/105097605-e33d0580-5a76-11eb-9103-ba9cce555c5b.png)

This is valid javadoc. Adding.

Reply:
Done.

Reply:
input code is always a text, javadoc as image is ok.